db/migrate/20201106150006_increase_max_length_of_avatar_url.rb

```diff
@@ -0,0 +1,5 @@
+class IncreaseMaxLengthOfAvatarUrl < ActiveRecord::Migration[5.2]
+  def change
+    change_column :users, :avatar_url, :string, :limit => nil
```

Comment:
You're using `change` but `change_column` doesn't know how to go backwards. Can you change it to `up` and `down`

Reply:
ahh, sure, and I guess I would put `change_column :users, :avatar_url, :string, :limit => 255` in the `down`?

Reply:
Yup

Reply:
OK, just tested and that doesn't work if there are any values longer than 255 in the table, so it's not very practically useful, but I can include it just for completenesses sake unless you have a better idea

Reply:
Eh, whatever. I don't think it matters. Leave it as is